lib/connections/relay_listen.go

```diff
@@ -71,13 +74,13 @@ func (t *relayListener) Serve() {
 
 			conn, err := client.JoinSession(inv)
 			if err != nil {
-				l.Infoln("Joining relay session (BEP/relay):", err)
+				l.Debugln("Listen (BEP/relay): failed to join session:", err)
 				continue
 			}
 
 			err = dialer.SetTCPOptions(conn)
 			if err != nil {
-				l.Infoln(err)
+				l.Debugln(err)
```

Comment:
Need context

Reply:
Actually I considered debug logging without context as ok, as the context can be established from file names and line numbers. I will add some (generic) context anyway.